CloudFormation/arkcase.yml

```diff
@@ -607,6 +609,79 @@ Resources:
         - Key: ManagedBy
           Value: CloudFormation
 
+  ###########################################################
+  # Task definition custom resource with EFS specifications #
+  ###########################################################
+
+  TaskDefinitionResourceLambdaExecutionRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service: lambda.amazonaws.com
+            Action: sts:AssumeRole
+      Policies:
+        - PolicyName: TaskDefinitionResourceLambaPolicy
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Action:
+                  - ecs:RegisterTaskDefinition
+                  - ecs:DeregisterTaskDefinition
+                Resource: "*"
+                Effect: Allow
+
+              - Action:
+                  - iam:GetRole
+                  - iam:PassRole
+                Resource: "*"
```

Comment:
Does this allow the lambda to assume any role?  If not, what does the `*` represent?